backend/src/schema/resolvers/registration.spec.js

```diff
@@ -307,6 +307,7 @@ describe('Signup', () => {
       it('is allowed to signup users by email', async () => {
         await expect(mutate({ mutation, variables })).resolves.toMatchObject({
           data: { Signup: { email: 'someuser@example.org' } },
+          errors: undefined,
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-09-30T08:59:23Z" title="Monday, September 30th 2019, 10:59:23 am +02:00">Sep 30, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -307,6 +307,7 @@ describe('Signup', () => {
       it('is allowed to signup users by email', async () => {
         await expect(mutate({ mutation, variables })).resolves.toMatchObject({
           data: { Signup: { email: 'someuser@example.org' } },
+          errors: undefined,
```


Is this necessary? I think `toMatchObject` does not need the full object defined, it should also pass for partial matches. Or do you want to explicitly make sure that `errors` are indeed `undefined`?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-01T22:49:19Z" title="Wednesday, October 2nd 2019, 12:49:19 am +02:00">Oct 2, 2019</time>_

---


Yes, I do. I thought it might increase readability and maybe save us from false negatives.